docs/shortcut-live-activity.md

```
@@ -0,0 +1,35 @@
+# Shortcuts → Live Activity
+
+#1222 asked for a Shortcut that starts a Live Activity for a stop and routes.
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)